src/servers/ZoneServer2016/entities/plantingdiameter.ts

```diff
@@ -46,4 +52,49 @@ export class PlantingDiameter extends TemporaryEntity {
     }
     return server.deleteEntity(this.characterId, server._temporaryObjects);
   }
+
+  OnPlayerSelect(server: ZoneServer2016) {
+    for (const clients in server._clients) {
```

Comment:
client is available as an arg of this callback so no need to iterate on all clients